pavelib/paver_tests/test_paver_quality.py

```diff
@@ -64,34 +64,6 @@ def test_pep8_parser(self):
         assert num == 2
 
 
-@ddt
-class TestPaverQualityOptions(unittest.TestCase):
-    """
```

Comment:
These tests were present to check that the pylint thresholds were being correctly read and parsed. After removing the thresholds, these are not needed anymore.